### PR TITLE
Removing UIWebView

### PR DESCRIPTION
**Pull request by Krumelur**
Replacing deprecated UIWebView with WKWebView

This fixes https://github.com/xamarin/XamarinUniversity/issues/1172

### CODE REVIEW

File: Exercise 1/Start/Clock/ViewControllers/AboutViewController.cs [hunk 1]
```diff
@@ -17,9 +18,24 @@ public override void ViewDidLoad ()
...
 
-			string aboutUrl = NSBundle.MainBundle.BundlePath + "/About.html";
+			var aboutUrl = NSUrl.FromFilename(NSBundle.MainBundle.BundlePath + "/About.html");
+			AboutWebView.NavigationDelegate = this;
+			AboutWebView.LoadFileUrl(aboutUrl, aboutUrl.RemoveLastPathComponent());
+		}
 
-			AboutWebView.LoadRequest (new NSUrlRequest (new NSUrl (aboutUrl, false)));
+		[Export("webView:decidePolicyForNavigationAction:decisionHandler:")]
+		public void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
```
Comment (patridge) [inline Exercise 1/Start/Clock/ViewControllers/AboutViewController.cs:27]: @Krumelur Do you mind walking me through this a little? Where does the `Action<WKNavigationActionPolicy>` parameter come from here?

**Review by patridge — CHANGES_REQUESTED** [review]
Thank you so much for putting this together; it looks great. I only had one question about it due to my lack of knowledge about WKWebView.

I do have one not-related-to-code action item. Do you mind removing the various .DS_Store files from Git and pushing up those removals? I will try to get that added to the .gitignore file for all the course repos, but we'll want to remove those files from here before then.

**Comment by Krumelur** [comment]
.DS_Store should be removed.

`WKWebView` allows navigation (it's a web view after all :-)) but by default this navigation would happen inside the view itself. The delegate method will make sure that the XamU link will be opened in the external browser. The delegate method is called before navigation is about to happen. The `Action<WKNavigationActionPolicy> decisionHandler` is a callback and a way of telling iOS if it should allow the navigation to happen or not. In case of XamU, we don't allow the (internal) navigation and perform a custom operation instead (open the browser). Does that help?

**Review by patridge — APPROVED** [review]

**Comment by patridge** [comment]
Confirmed changes run happily for Ex1/Completed and verified changes match up on Ex2-3.

**Pull request merged**